docs/gateway-api-migration-guide.md

```
@@ -0,0 +1,389 @@
+
+---
+
+## Part 1: For All Workbenches - General Requirements
+
+These requirements apply **regardless of whether you use nginx or application-level path handling**.
+
+### 1. Health Check Endpoints
+
+Your workbench **must** respond to health checks at:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)